src/mlx/warnings.py

```diff
@@ -53,17 +54,19 @@ def activate_checker(self, checker):
 
     def activate_checker_name(self, name):
         '''
-        Activate checker by name
+        Activates checker by name
 
         Args:
             name (str): checker name
+
+        Returns:
+            WarningsChecker: activated checker object, or None when no checker with the given name exists
         '''
         for checker in self.public_checkers:
             if checker.name == name:
                 self.activate_checker(checker)
-                break
-        else:
```

Comment:
uhhh i loved `for-else` here.

Reply:
It's no longer needed, but I readded it to improve readibility: 9fe4830aa88d9dae48b96e0ea5021558d06b8b80